bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/api/dto/clip2/OnState.java

```diff
@@ -38,7 +38,8 @@ public boolean isOn() throws DTOPresentButEmptyException {
         throw new DTOPresentButEmptyException("'on' DTO is present but empty");
     }
 
-    public void setOn(boolean on) {
+    public OnState setOn(boolean on) {
```

Comment:
Can you revert this change since it's no longer needed?

Reply:
I think it is better to keep it, as almost every other setter method already returns `this` .. and this particular seter was an odd exception.

Reply:
Okay, let's keep it for consistency with existing pattern.